sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -8,8 +8,10 @@
 - Fixes an issue where non-retryable errors caused by a connection disconnecting were not getting surfaced to the user's registered error handler
   when using the `registerMessageHandler` method on a receiver.
   [PR 8401](https://github.com/Azure/azure-sdk-for-js/pull/8401)
-- Updates the way connection disconnects are handled to create a new connection object rather than re-using the existing one.
-  This should reduce some issues caused when the existing connection gets into an unexpected state.
+- Fixes reconnection issues by creating a new connection object rather than re-using the existing one.
+  [PR 8447](https://github.com/Azure/azure-sdk-for-js/pull/8447)
+- Adds a new method `open()` on the sender to allow you to front load the work of setting up the underlying AMQP links. Use this if you want to avoid having your first `send()` operation pay the tax of link set up.
+  [PR 8329](https://github.com/Azure/azure-sdk-for-js/pull/8329). This PR also fixes a bug where a sender recovering from connection loss does not report the error back to the user from ongoing send operations.
```

Comment:
Sorry! The issue was not that the error was not reported, it was tht it would get reported after the entire detached() completes which could take 3 hours :(

```suggestion
  [PR 8329](https://github.com/Azure/azure-sdk-for-js/pull/8329). This PR also fixes a bug where a sender recovering from connection loss does not report the error back to the user from ongoing send operations in expected time.
```